src/linechart/VoronoiCircleContainer.jsx

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var React = require('react');
+var d3 = require('d3');
+var shade = require('../utils').shade;
+var VoronoiCircle = require('./VoronoiCircle');
+
+module.exports = React.createClass({
+
+  displayName: 'VornoiCircleContainer',
+
+  getDefaultProps() {
+    return { 
+      circleRadius: 3,
+      circleFill: '#1f77b4',
+      hoverAnimation: true
+    };
+  },
+
+  getInitialState() {
+    return { 
+      circleRadius: this.props.circleRadius,
+      circleFill: this.props.circleFill
+    }
+
+  },
+
+  render() {
+
+    var props = this.props;
+
+    // animation controller
+    var handleMouseOver, handleMouseLeave;
+    if(props.hoverAnimation) {
+      handleMouseOver = this._animateCircle;
+      handleMouseLeave = this._restoreCircle;
+    } else {
+      handleMouseOver = handleMouseLeave = null;
+    }
+
+    return (
+      <g>
+        <VoronoiCircle
+            handleMouseOver={handleMouseOver}
+            handleMouseLeave={handleMouseLeave}
+            voronoiPath={this._drawPath(props.vnode)}
+            cx={props.cx}
+            cy={props.cy}
+            circleRadius={this.state.circleRadius}
+            circleFill={this.state.circleFill}
+        />
+      </g>
+    );
+  },
+
+  _animateCircle() {
```

Comment:
I again think these numbers should be props, so the user has control over the animation (not to mention that again I don't actually think we should do animation).